aws-s3-transfer-manager/examples/cp.rs

```diff
@@ -278,6 +281,7 @@ async fn main() -> Result<(), BoxError> {
         tracing_subscriber::fmt()
             .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
             .with_thread_ids(true)
+            .with_ansi(false) // Disable ANSI colors
```

Comment:
question: why?

Reply:
with the ansi color, it's unreadable from pure text. I think it will be easier to read and parse.